src/common/ActivityInfo/ActivityInfo.tsx

```diff
@@ -48,6 +73,64 @@ export const ActivityInfo: React.FC<InfoProps> = (props: InfoProps) => {
   //    */
   // };
 
+  const getICS = () => {
```

Comment:
- Use React.useCallback
- Add comments to function to make it more obvious what it does.